src/main/java/com/google/sps/servlets/GetBlurAreasServlet.java

```diff
@@ -214,4 +270,22 @@ private void deleteFile(BlobKey blobKey) {
     BlobstoreService blobstoreService = BlobstoreServiceFactory.getBlobstoreService();
     blobstoreService.delete(blobKey);
   }
+
+  /**
+   * @param image contains the Image object for which to create our annotation.
+   * @param type contains the feature type of our annotation.
+   */
+  private AnnotateImageRequest createAnnotateImageRequest(Image image, Feature.Type type) {
+    // Setting max results to 0 disables the upper limit.
+    Feature feature = Feature.newBuilder().setType(type).setMaxResults(0).build();
+    return AnnotateImageRequest.newBuilder().addFeatures(feature).setImage(image).build();
```

Comment:
Isn't it possible to create one request with all the features? I mean something like
AnnotateImageRequest.newBuilder().addFeatures(feature1).addFeatures(feature2);...